api/api_full.go

```diff
@@ -850,6 +863,10 @@ type ChannelAvailableFunds struct {
 	ConfirmedAmt types.BigInt
 	// PendingAmt is the amount of funds that are pending confirmation on-chain
 	PendingAmt types.BigInt
+	// AvailableAmt is part of ConfirmedAmt that is available for use (pre-allocated)
+	AvailableAmt types.BigInt
+	// PendingAvailableAmt is the amount of available funds that are pending confirmation on-chain
```

Comment:
To clarify this is a portion of the "PendingAmt" not the "AvailableAmt"

```suggestion
// PendingAvailableAmt is the amount of funds that are pending confirmation on-chain that will become available once confirmed
```